code/game/objects/effects/effect_system/effects_smoke.dm

```diff
@@ -218,3 +218,42 @@
 
 /datum/effect_system/smoke_spread/transparent
 	effect_type = /obj/effect/particle_effect/smoke/transparent
+
+/////////////////////////////////////////////
+// Changeling smoke
+/////////////////////////////////////////////
+
+/obj/effect/particle_effect/smoke/changeling
+	color = "#7A61B0"
+	lifetime = 10
+
+/obj/effect/particle_effect/smoke/changeling/smoke_mob(mob/living/carbon/M)
+	if(!..())
+		return
+
+	var/datum/antagonist/changeling/C = M.mind?.has_antag_datum(/datum/antagonist/changeling)
+	if(C && C.get_ability(/datum/action/changeling/miasmic_mist))
+		return
+
+	M.drop_item()
+	M.Jitter(10 SECONDS)
+	M.AdjustEyeBlurry(10 SECONDS)
+	M.AdjustDizzy(5 SECONDS)
+	M.AdjustSlur(10 SECONDS)
+	M.emote(pick("drool", "quiver", "tremble", "scream"))
+	M.fakevomit()
+	M.adjustToxLoss(2)
+	M.drop_l_hand()
```

Comment:
Why drop_item(), drop_l_hand(), and drop_r_hand? Doesn't the cough emote auto stop stuff too?